src/gtimelog/main.py

```diff
@@ -1342,12 +1342,12 @@ def on_tasks_file_changed(self, monitor, file, other_file, event_type):
             GLib.timeout_add_seconds(1, self.check_reload_tasks)
 
     def check_reload(self):
-        if self.timelog.check_reload():
+        if self.timelog and self.timelog.check_reload():
             self.notify('timelog')
             self.tick(True)
 
     def check_reload_tasks(self):
-        if self.tasks.check_reload():
+        if self.tasks and self.tasks.check_reload():
```

Comment:
I've just seen this change in the other PR, haven't I?

Reply:
Yes. My mistake. I think I just forgot to go back to master before starting the new branch.